src/persistReducers.js

```diff
@@ -44,7 +46,7 @@ export default function persistReducer(config, baseReducer) {
     state.has('_persist') && state.get('_persist').get('rehydrated') &&
       _persistoid &&
       !_paused &&
-      _persistoid.update(state)
+      _persistoid.update(state.toJS())
```

Comment:
from now it can be handled by redux-persist persistoid. presistoid do not have to be aware of immutability